ocpp/v20/enums.py

```diff
@@ -0,0 +1,697 @@
+    local_reset = "LocalReset"
+    power_up = "PowerUp"
+    remote_reset = "RemoteReset"
+    scheduled_reset = "ScheduledReset"
+    triggered = "Triggered"
+    unknown = "Unknown"
+    watchdog = "Watchdog"
+
+
+class CertificateStatus:
+    accepted = "Accepted"
+    signature_error = "SignatureError"
+    certificate_expired = "CertificateExpired"
+    certificate_revoked = "CertificateRevoked"
+    no_certificate_available = "NoCertificateAvailable"
+    cert_chain_error = "CertChainError"
+    contract_cancelled = "ContractCancelled"
+
+
+class ChargingLimitSource:
+    ems = "EMS"
+    other = "Other"
+    so = "SO"
+    cso = "CSO"
+
+
+class ChargingProfileKind:
+    absolute = "Absolute"
+    recurring = "Recurring"
+    relative = "Relative"
+
+
+class ChargingProfilePurpose:
+    charging_station_external_constraints =\
+        "ChargingStationExternalConstraints"
+    charging_station_max_profile = "ChargingStationMaxProfile"
+    tx_default_profile = "TxDefaultProfile"
+    tx_profile = "TxProfile"
+
+
+class ChargingRateUnit:
+    w = "W"
+    a = "A"
+
+
+class ChargingState:
+    charging = "Charging"
+    ev_detected = "EVDetected"
+    suspended_ev = "SuspendedEV"
+    suspended_evse = "SuspendedEVSE"
+
+
+class ClearMonitoringStatus:
+    accepted = "Accepted"
+    rejected = "Rejected"
+    not_found = "NotFound"
+
+
+class ComponentCriterion:
+    active = "Active"
+    available = "Available"
+    enabled = "Enabled"
+    problem = "Problem"
+
+
+class Connector:
+    c_ccs1 = "cCCS1"
+    c_ccs2 = "cCCS2"
+    c_g105 = "cG105"
+    c_tesla = "cTesla"
+    c_type1 = "cType1"
+    c_type2 = "cType2"
+    s309_1p_16a = "s309-1P-16A"
+    s309_1p_32a = "s309-1P-32A"
+    s309_3p_16a = "s309-3P-16A"
+    s309_3p_32a = "s309-3P-32A"
+    s_bs1361 = "sBS1361"
+    s_cee_7_7 = "sCEE-7-7"
+    s_type_2 = "sType2"
+    s_type_3 = "sType3"
+    other1_ph_max_16a = "Other1PhMax16A"
+    other1_ph_over_16a = "Other1PhOver16A"
+    other3_ph = "Other3Ph"
+    pan = "Pan"
+    w_inductive = "wInductive"
+    w_resonant = "wResonant"
+    undetermined = "Undetermined"
+    unknown = "Unknown"
+
+
+class CostKind:
+    carbon_dioxide_emission = "CarbonDioxideEmission"
+    relative_price_percentage = "RelativePricePercentage"
+    renewable_generation_percentage = "RenewableGenerationPercentage"
+
+
+class Data:
+    string = "string"
+    decimal = "decimal"
+    integer = "integer"
+    date_time = "dateTime"
+    boolean = "boolean"
+    option_list = "OptionList"
+    sequence_list = "SequenceList"
+    member_list = "MemberList"
+
+
+class EncodingMethod:
+    other = "Other"
+    dlms_message = "DLMS Message"
+    cosem_protected_data = "COSEM Protected Data"
+    edl = "EDL"
+
+
+class EnergyTransferMode:
+    ac_single_phase_core = "AC_single_phase_core"
+    ac_three_phase_core = "AC_three_phase_core"
+    dc_combo_core = "DC_combo_core"
+    dc_core = "DC_core"
+    dc_extended = "DC_extended"
+    dc_unique = "DC_unique"
```

Comment:
<img width="564" alt="Bildschirmfoto 2020-05-21 um 11 49 23" src="https://user-images.githubusercontent.com/12515990/82546885-2433f180-9b59-11ea-9fd1-32ef6a658952.png">

You are probably getting the enums from OCPP 2.0, but that version is deprecated and we should use 2.0.1 instead